src/pages/Home.jsx

```diff
@@ -0,0 +1,26 @@
+import { Categories } from "components/Home"
+import { Header } from "components/Reusable"
+import { useTheme } from "contexts"
+import { getBgColor, getTextColor } from "utils"
+
+const Home = () => {
+    const { theme } = useTheme()
+
+    return (
+        <div
+            style={{
```

Comment:
If not necessary try not to use inline styles

Reply:
Sure, will change it.